src/Loading/Plugins/babylon.babylonFileLoader.ts

```diff
@@ -478,7 +478,12 @@
                 camera.target = BABYLON.Vector3.FromArray(parsedCamera.target);
             }
         } else {
-            camera.rotation = BABYLON.Vector3.FromArray(parsedCamera.rotation);
+            if (parsedCamera.rotationQuaternion) {
```

Comment:
Warning here: only the followCamera uses the rotationQuaternion. All others cameras only use euler rotation